.run/kuksa-vss-processor-Plugin_FunctionalTests.run.xml

```diff
@@ -0,0 +1,23 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
We should try to not create to many run configurations for the tests, because let's be honest most of them won't be used anyway after initial testing. in the rare case that we want to use them or have to use them they can be run via command line. In the authentication ticket I also removed the Secure stuff and replaced it by a small comment on how to run the tests using gradlew. 

As long as the tests are run when executing ./gradlew clean build resp. ./gradlew clean test this would be sufficient for me. 

We probably should make sure that the special test cases are run on the CICD however.

Reply:
Removed the run config